app/src/main/java/io/neurolab/main/DeviceConnector.java

```diff
@@ -0,0 +1,101 @@
+package io.neurolab.main;
+
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.hardware.usb.UsbDevice;
+import android.hardware.usb.UsbDeviceConnection;
+import android.hardware.usb.UsbManager;
+import android.util.Log;
+
+import com.felhr.usbserial.UsbSerialDevice;
+import com.felhr.usbserial.UsbSerialInterface;
+
+import java.io.UnsupportedEncodingException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DeviceConnector {
+
+    UsbManager usbManager;
+    UsbDevice device;
```

Comment:
declare them private and use getters and setters

Reply:
not all of these members need to be exposed